src/state/farms/hooks.ts

```diff
@@ -54,7 +54,7 @@ export const usePollFarmsWithUserData = () => {
     : ['farmsWithUserData', account, chainId]
 
   useSWRImmutable(
-    account && chainId ? name : null,
+    account && chainId && !isProxyContractLoading ? name : null,
```

Comment:
For not boosted farms, does this logic still fetch `fetchFarmUserDataAsync`, right?

Reply:
Got it. Thanks for your contribution.